e2e/smoke/workspace_status.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# This serves as an example of how the workspace_status.sh script can be used to
+# stamp the `created` attribute of an image.
+BUILD_TIMESTAMP=${BUILD_TIMESTAMP:-$(date +%s)} # macOS specific
```

Comment:
e2e/smoke is run on multiple platforms, including linux and macos. Can we simplify this so it's not platform specific?

Reply:
We sure can, note though that in fact this file isn't actually being used (since CI doesn't --stamp). That's why I just added the comment above. I have no strong feelings about that file, I might as well remove it.